vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -207,7 +207,7 @@ def save_network_inventory(guest_device, hash)
   def save_networks_inventory(hardware, hashes, mode = :refresh)
     return if hashes.nil?
 
-    deletes = hardware.networks(true).dup
+    deletes = hardware.networks(true).to_a.dup
```

Comment:
The rest are covered by a `to_a` in `save_inventory_multi`; this one manipulates the result before it gets there.